### PR TITLE
Update open vm tools

### PR DESCRIPTION
**Pull request by dav1x**
1. Update the config.json template for system containers.

### CODE REVIEW

**Comment by yuqi-zhang** [comment]
bot, add author to whitelist

File: open-vm-tools-centos/config.json.template [hunk 1]
```diff
@@ -321,13 +477,13 @@
...
                 {
-                    "allow": false,
+                    "allow": true,
                     "access": "rwm"
                 }
             ]
         },
         "namespaces": [
-	    {
+        {
```
Comment (yuqi-zhang) [inline open-vm-tools-centos/config.json.template]: Could you align this as well

File: open-vm-tools-centos/config.json.template [hunk 2]
```diff
@@ -241,23 +399,21 @@
...
+            "source": "/dev/rtc",
             "options": [
-                "rbind",
                 "rw",
-                "mode=755"
+                "rbind"
             ]
         },
         {
+            "destination": "/proc",
```
Comment (yuqi-zhang) [inline open-vm-tools-centos/config.json.template]: It seems that /hostproc and /proc is bound earlier in the file, is there a reason to do this (with no options)?

File: open-vm-tools-centos/config.json.template [hunk 3]
```diff
@@ -13,49 +13,207 @@
         "env": [
             "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
             "TERM=xterm",
-            "NAME=$NAME",
-	    "SYSTEMD_IGNORE_CHROOT=1"
+            "NAME=open-vm-tools",
+        "SYSTEMD_IGNORE_CHROOT=1"
```
Comment (yuqi-zhang) [inline open-vm-tools-centos/config.json.template]: Please align

**Comment by dav1x** [comment]
Closing this. 

Please see https://github.com/projectatomic/atomic-system-containers/pull/90

**Pull request closed without merging**